projects/storefrontlib/src/cms-components/product/product-images/product-images.component.ts

```diff
@@ -30,10 +32,15 @@ export class ProductImagesComponent {
     map(([, container]) => container)
   );
 
+  // Used for image zoom
+  expandImage = new BehaviorSubject(false);
```

Comment:
`expendImage` is confusing. It is not `image` but whether the image is expended, so may be `isImageExpended$` or `isImageDialogOpened$` or `isImageZoomed$`... or whatever make sense :)